.gitignore

```diff
@@ -102,3 +102,5 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+oscar/
```

Comment:
Ignore oscar?????

Reply:
oscar folder is vitualenv

Reply:
You should ignore all in venv folder, not just oscar ja